lib/errors.js

```diff
@@ -5,13 +5,24 @@
 // lib
 import { HTTP_STATUS_CODE } from "./fetch-request";
 
+/**
+ * Log the error to the console with a timestamp.
+ * @param {string|number} statusCode Error status code, e.g. 403
+ * @param {string} title Narrative string corresponding to error code
+ * @returns {void}
+ */
+export function logError(statusCode, title) {
+  const date = new Date().toISOString();
+  console.log(`ERROR [${date}] ${statusCode}:${title}`);
```

Comment:
Should this be `console.error` instead of `log`? `Error` will print to standard error instead of standard out. Is that a distinction on web pages?

Reply:
AFAIK there’s no distinction in the server logs. Browsers can display them in different colors, but I don’t think anything beyond that.

Reply:
I’ll leave them as is. We’ve had other error messages that use console.log, so if we decide they should use `console.error`, then we can convert all of them at once.